compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs

```diff
@@ -240,6 +291,35 @@ impl Ssa {
             if let Some(mut generated_acir) =
                 context.convert_ssa_function(&self, function, brillig)?
             {
+                // We want to be able to insert Brillig stdlib functions anywhere during the ACIR generation process (e.g. such as on the `GeneratedAcir`).
+                // As we do want a reference to the `SharedContext` on the generated ACIR itself,
+                // we instead store the opcode location at which a Brillig call to a std lib function occurred.
```

Comment:
```suggestion
                // we instead store the opcode location at which a Brillig call to a stdlib function occurred.
```

Reply:
I do separate std lib by a space in multiple locations. You prefer we did `stdlib` ?

Reply:
I'm going to resolve this conversation for now. If we want to edit the comments we can have a chore PR later